### PR TITLE
Add Filmora Video Editor 10.5 

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19761)

### CODE REVIEW

File: manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: "WondershareSoftwareCoLtd.WondershareFilmora"
+PackageVersion: "10.5"
+PackageLocale: "en-US"
+Publisher: "Wondershare Software Co. Ltd."
+PackageName: "Wondershare Filmora"
+License: "EULA"
```
Comment (quhxl) [inline manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml:6]:
```suggestion
License: https://www.wondershare.com/company/end-user-license-agreement.html
```

**Review by quhxl — CHANGES_REQUESTED** [review]

File: manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: "WondershareSoftwareCoLtd.WondershareFilmora"
+PackageVersion: "10.5"
+PackageLocale: "en-US"
+Publisher: "Wondershare Software Co. Ltd."
```
Comment (quhxl) [inline manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml:4]:
```suggestion
Publisher: Wondershare Software
```
ARP

**Review by quhxl — CHANGES_REQUESTED** [review]

File: manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml (hunk 2 above)
Comment (vedantmgoyal2009) [reply manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml:4]: no

Comment (vedantmgoyal2009) [reply manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml:4]: publisher is correct, as seen in official website

File: manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: "WondershareSoftwareCoLtd.WondershareFilmora"
```
Comment (OfficialEsco) [inline manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml:1]:
```suggestion
PackageIdentifier: "Wondershare.Filmora"
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
This would also looks a lot better, remember to rename the folder and yaml file.

File: manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml (hunk 2 above)
Comment (OfficialEsco) [reply manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml:4]: > publisher is correct, as seen in official website

image

Its not.

**Comment by vedantmgoyal2009** [comment]
@OfficialEsco  please approve my changes

**Comment by vedantmgoyal2009** [comment]
@OfficialEsco @msftbot @wingetbot @quhxl  Someone please approve my pull request

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Your PR will not get accepted faster just because you're spamming.
You don't listen to Suggested Changes https://github.com/microsoft/winget-pkgs/pull/19761#discussion_r663496567.
And i've noticed you lied about these.
```
[X] Have you validated your manifest locally with winget validate --manifest <path>?
[X] Have you tested your manifest locally with winget install --manifest <path>?
```

File: manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml (hunk 2 above)
Comment (quhxl) [reply manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml]: Thanks @OfficialEsco! The PackageName, Publisher and the PackageVersion must be the same as in the uninstall list. (ARP)

File: manifests/w/Wondershare/Filmora/10.5/Wondershare.Filmora.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: "Wondershare.Filmora"
+PackageVersion: "10.5"
+PackageLocale: "en-US"
+Publisher: "Wondershare Software Co. Ltd."
+PackageName: "Wondershare Filmora"
+License: https://www.wondershare.com/company/end-user-license-agreement.html
```
Comment (quhxl) [inline manifests/w/Wondershare/Filmora/10.5/Wondershare.Filmora.yaml:6]:
```suggestion
License: Copyright © 2021 Wondershare.
LicenseUrl: https://www.wondershare.com/company/end-user-license-agreement.html
```
My mistake. Can you please accept this?

**Review by quhxl — CHANGES_REQUESTED** [review]

File: manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml (hunk 2 above)
Comment (vedantmgoyal2009) [reply manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml]: sorry, here is has been written "Wondershare Software Co. Ltd." [in pdfelement installation]
image

Comment (vedantmgoyal2009) [reply manifests/w/WondershareSoftwareCoLtd/WondershareFilmora/10.5/WondershareSoftwareCoLtd.WondershareFilmora.yaml]: changes made...

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**